wpimath/src/main/native/include/wpi/math/kinematics/SwerveModuleAccelerations.hpp

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#pragma once
+
+#include <wpi/util/SymbolExports.hpp>
```

Comment:
Replace all instances of angle brackets in subproject includes with double quotes. Headers from other allwpiilb subprojects are now considered project headers instead of thirdparty library headers.